go-runtime/compile/build.go

```diff
@@ -293,7 +294,33 @@ var scaffoldFuncs = scaffolder.FuncMap{
 	},
 	"valueEnum": func(d schema.Decl) bool {
 		if e, ok := d.(*schema.Enum); ok {
-			if len(e.Variants) > 0 && e.Variants[0].Value != nil {
+			return e.IsValueEnum()
```

Comment:
nit: i think you can actually call this directly from the template! 
something like this:
```
{{if and (is "Enum" .) .IsValueEnum }}
...
{{else if is "Enum" . }}
```

Reply:
Ooo thanks for the reminder! So much nicer now :D